usage.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: usage
+title: Quarkus Usage Analytics
+subtitle: 
+permalink: /usage/
+---
+
+Starting from Quarkus 3.2 when you run Quarkus the first time in dev mode (`quarkus dev`, `mvn quarkus:dev`, etc.) you get asked if you agree to contribute anonymous build data to the Quarkus community.
+
+![/assets/images/usage-prompt.png](/assets/images/usage-prompt.png)
+
+If you answer Yes then, when you perform a quarkus build, anonymized data is sent
+to gather usage statistics on how Quarkus is being used and adopted.
```

Comment:
I think this shouldn't be on a new line. Not a biggy, but makes it harder to review and propose changes.